2-mandatory/2-oxygen-levels.js

```diff
@@ -11,7 +11,15 @@
     Some string methods that might help you here are .replace() and .substring().
 */
 
-function findSafeOxygenLevel() {}
+function checkLevelOxygen(oxygen) {
+  let oxygenValue = parseFloat(oxygen.replace("%", ""));
+  return oxygenValue > 19.5 && oxygenValue < 23.5 && oxygen.includes("%");
```

Comment:
You handle this well by converting the string to a float, and then checking both.